lib/modules/platform/types.ts

```diff
@@ -28,6 +28,7 @@ export interface PlatformResult {
 export interface RepoResult {
   defaultBranch: string;
   isFork: boolean;
+  repoCacheId: string;
```

Comment:
```suggestion
  repoId: string;
```

The platform layer isn't concerned with whether this ID is for cache or other reasons

Reply:
Now I think it's better to call it `fingerprint` or `repoFingerprint`, as we probably want to distinguish not only `id` but also `endpoint`

Reply:
sound good